content/_data/logo/general.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: 'General'
```

Comment:
There are two versions of the general:

* original red-faced one
* pale-faced one, which is being used on stickers. It was a patch by @KostyaSha AFAIK

Not sure how the attribution should be done in this case